framework/src/preconditioners/MoosePreconditioner.C

```diff
@@ -27,7 +27,7 @@ InputParameters validParams<MoosePreconditioner>()
   InputParameters params = validParams<MooseObject>();
   params.addPrivateParam<FEProblemBase *>("_fe_problem_base");
 
-  MooseEnum pc_side("left right symmetric", "right");
+  MooseEnum pc_side("left right symmetric");
```

Comment:
`default` is better than `none`. The former actually means use whatever PETSc thinks, while the latter implies "don't do anything".